src/main/java/frc/robot/commands/ShootNoteCommand.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.ScoringConstants;
+import frc.robot.subsystems.Scoring;
+
+public class ShootNoteCommand extends Command {
+    private Scoring scoring = Scoring.getInstance();
+    private Timer timer = new Timer();
+    private int secondsToScore;
+
+    public ShootNoteCommand(int secondsToScore) {
+        this.addRequirements(scoring);
+        this.secondsToScore = secondsToScore;
+    }
+
+    public ShootNoteCommand() {
+        this.addRequirements(scoring);
+    }
```

Comment:
In this constructor, set the "secondsToScore" instance equal to the default constant.

Reply:
I disagree with this comment. The command is currently set up where if seconds to shoot is not provided, then isFinished returns false (most likely for operator control).
Might be good to add a comment explaining this behavior.